src/ebay-menu/types.ts

```diff
@@ -1,2 +1,28 @@
+import { ComponentProps, Ref } from 'react'
+import { EbayEventHandler, EbayKeyboardEventHandler } from '../common/event-utils/types'
+
 export type EbayMenuType = 'radio' | 'checkbox'
 export type EbayMenuPriority = 'primary' | 'secondary' | 'none'
+
+type SpanProps = Omit<ComponentProps<'span'>, 'onKeyDown' | 'onChange'>
+type SelectProps = {
+    index: number;
+    checked: number[];
+}
+type ChangeProps = SelectProps & {
+    indexes: number[];
```

Comment:
why do we return the indexes as well?

Reply:
I guess the idea was to return indexes that are checked in case of `radio` or `checkbox` type of the EbayMenu.
That's the original EbayUI API that was implemented in Marko.